packages/common/src/access/can-edit-dataset.ts

```diff
@@ -0,0 +1,13 @@
+import { IItem, IUser } from "@esri/arcgis-rest-types";
+import { hasBasePriv } from "./has-base-priv";
+import { canEditItem } from "./can-edit-item";
+
+/**
+ *
```

Comment:
Please add a description for this method. In particular what it means to check for editing.